src/main/grammars/bitbucket/Codeowners.bnf

```diff
@@ -60,7 +60,7 @@ TeamDefinition          ::= '@''@''@' TeamName WSS Owners {
   mixin = "com.github.fantom.codeowners.lang.kind.bitbucket.psi.impl.CodeownersTeamDefinitionExtImpl"
 }
 
-Assign                  ::= Pattern /*SPACES*/ WSS Owners /*CRLF*/
+Assign                  ::= Pattern /*SPACES*/ WSS Owners WSS? /*CRLF*/
```

Comment:
Actually, `Assign` is a part of `Rule` which is part of `value_item_` which allows trailing spaces, so need to understand why they are still not allowed

Reply:
Did you check that it works? Because the issue is that when there are spaces after the `Owner`, parser expected other owners, which are not here. And since it happens during parsing the `Owners` rule, I'd expect it wouldn't match trailing spaces in `Assign` rule